docs/creating-your-site.md

```diff
@@ -1,16 +1,29 @@
 # Creating your site
 
-After you've [installed] Material for MkDocs, you can bootstrap your project
-documentation using the `mkdocs` executable. Go to the directory where you want
-your project to be located and enter:
+After you've [installed] Material for MkDocs, you can bootstrap your documentation project then preview your documentation and finaly build your site using the `mkdocs` executable.
```

Comment:
Could you explain why this change is necessary? It doesn't have anything to do with what you described in your PR.